### PR TITLE
Fixed editing recordings, and recording only new episodes

### PR DESCRIPTION
**Pull request by sub3**
-Fixed issue where user couldn't select to only record new episodes
-Fixed issue with updating one-off recordings
-Fixed issue where pre/post padding was not being loaded with recording list

### CODE REVIEW

**Comment by ksooo** [comment]
Could you please bump the addon version and update the changelog?

File: src/pvrclient-nextpvr.cpp [hunk 1]
```diff
@@ -1199,16 +1208,31 @@ PVR_ERROR cPVRClientNextPVR::GetTimers(ADDON_HANDLE handle)
         if (pRecordingNode->FirstChildElement("recurring_parent") != NULL)
         {
           tag.iParentClientIndex = atoi(pRecordingNode->FirstChildElement("recurring_parent")->FirstChild()->Value());
-          if (tag.iTimerType == TIMER_ONCE_EPG)
-          {
-            tag.iTimerType = TIMER_ONCE_EPG_CHILD;
-          }
-          else
+          if (tag.iParentClientIndex != 0)
```
Comment (ksooo) [inline src/pvrclient-nextpvr.cpp]: Please use PVR_TIMER_NO_PARENT instead of a 'magic number'.

**Comment by ksooo** [comment]
BTW: I reverted the Jarvis commit you did by "accident".

Please open a Jarvis PR after(!) we're done with this PR. Please, in case you are uncertain how to do this right, without accidentally pushing stuff directly to the repo, feel free to ask. I suggest reading https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md, which is pretty much identical to how it works for pvr addons.

**Comment by sub3** [comment]
I've corrected the items version/changelog, and updated to use PVR_TIMER_NO_PARENT

**Comment by ksooo** [comment]
Cannot say anything about the nextpvr specific code. Otherwise it is looking good. Thanks.

**Pull request merged**